app/generator/content/schedule/part3/get-table.js

```diff
@@ -1,6 +1,8 @@
 const toCurrencyString = require('../../../../generator/to-currency-string')
```

Comment:
Need to rely whether the grey background for the table and payment summary is correct as template does not have any background to it.

Reply:
Background has been removed.